### ISSUE TITLE
Finn doesn't give strain when used defensively

### ISSUE THREAD

**Issue by chrisalleng** [punkUser/xwing_math#24]
https://github.com/punkUser/xwing_math/blob/376f55777be097a54d17a74573b7949ea045bfb6/source/modify_defense_dice.d#L658

I didn't check offensively.  Also, it's time to just search and see if adding a blank or focus is better ;)

**Comment by punkUser** [comment]
Yeah this is known/by design (strain is not implemented currently), but good to have an issue to track it.
